test/integration/helpers/Npm.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+
+namespace Appium.Net.Integration.Tests.helpers
+{
+    internal class Npm
+    {
+
+        public static string GetNpmPrefixPath()
+        {
+            string npmPath = GetNpmExecutablePath();
+            string npmPrefixPath = RunCommand(npmPath, "-g root");
+
+            return npmPrefixPath.Trim();
+        }
+
+        private static string RunCommand(string command, string arguments, int timeoutMilliseconds = 30000)
+        {
+            try
+            {
+                using (Process process = new Process
+                {
+                    StartInfo = new ProcessStartInfo
+                    {
+                        FileName = command,
+                        Arguments = arguments,
+                        RedirectStandardOutput = true,
+                        RedirectStandardError = true,
+                        UseShellExecute = false,
+                        CreateNoWindow = true,
+                    }
+                })
+                {
+                    process.Start();
+                    string output = process.StandardOutput.ReadToEnd();
+                    string errorOutput = process.StandardError.ReadToEnd();
+                    _ = process.WaitForExit(timeoutMilliseconds);
+
+                    int exitCode = process.ExitCode;
+                    if ((exitCode == 1) && command.Contains("npm"))
+                    {
+                        Console.WriteLine($"npm Error upon command: `{arguments}`. {output}");
+                        throw new NpmUnknownCommandException($"Command: `{arguments}` exited with code {exitCode}. Error: {output}");
+                    }
+
+                    return output;
+                }
+            }
+            catch (Win32Exception ex) when (command.Contains("npm"))
```

Comment:
Should we enclose the whole method body into try/catch? I believe this exception is only thrown upon some particular API call. I usually try to minimise exception blocks to a very necessary

Reply:
I can basically wrap the code up until `process.Start()`